lib/mitie/training_instances/ner.rb

```diff
@@ -0,0 +1,51 @@
+module Mitie
+  module TrainingInstances
+    class NER
+      def initialize(tokens)
+        tokens_pointer = Utils.array_to_pointer(tokens)
+
+        @pointer = FFI.mitie_create_ner_training_instance(tokens_pointer)
+        raise "Unable to create ner_training_instance. Probably ran out of RAM." unless @pointer
```

Comment:
`@pointer` should always be a `Fiddle::Pointer`, so think this needs to be `if @pointer.null?`